src/player/CMakeLists.txt

```diff
@@ -71,7 +77,7 @@ endif()
 
 if(CORRADE_TARGET_EMSCRIPTEN)
     # globals yay :/
-    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -s 'EXTRA_EXPORTED_RUNTIME_METHODS=[\"ccall\"]' -s ALLOW_MEMORY_GROWTH=1")
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -s EXTRA_EXPORTED_RUNTIME_METHODS=[\"ccall\"] -s ALLOW_MEMORY_GROWTH=1")
```

Comment:
Did this cause a problem on Windows or why is this?

Reply:
It caused a compile error, yes. I suppose it has something to do with the `"` in the value...